### DISCUSSION TITLE
Add Ken Foster to the Nebari Triage team

### DISCUSSION THREAD

**Discussion by pavithraes** [nebari-dev/nebari#2109] [Community]
Ken (@kenafoster) has made some [helpful contributions to Nebari](https://github.com/nebari-dev/nebari/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Amerged+author%3Akenafoster) and will be working on a new extension to enhance the user sign-up workflows. I nominate him to join the [Nebari Triage team](https://github.com/orgs/nebari-dev/teams/triaging) to support his upcoming contributions. :)

Pinging @nebari-dev/core-team  - to share any thoughts and add Ken if folks agree.

**Comment by kcpevey** [comment]
I agree!

**Comment by dharhas** [comment]
approved & added
